pkg/bootstrap/bootstrap_config.go

```diff
@@ -343,6 +344,17 @@ func WriteBootstrap(config *meshconfig.ProxyConfig, node string, epoch int, pilo
 	// Support multiple network interfaces
 	meta[model.NodeMetadataInstanceIPs] = strings.Join(nodeIPs, ",")
 
+	if sdsEnabled {
+		opts["sds_enabled"] = "true"
```

Comment:
Remove those please. 
Instead of sds_enabled - we should use model.NodeMetaSdsUDSPAth != "" 
It doesn't matter if it was enabled or not if the path is missing.